src/style.css

```
@@ -0,0 +1,139 @@
+/* #region page */
+.page {
```

you can use just `html` for setting styles
```suggestion
html {
```